src/components/application-loader/initializers/configLoader.ts

```diff
@@ -6,13 +6,15 @@
 
 import { getConfig } from '../../../api/config'
 import { setApiUrl } from '../../../redux/api-url/methods'
-import { setBanner } from '../../../redux/banner/methods'
 import { setConfig } from '../../../redux/config/methods'
 import { getAndSetUser } from '../../login-page/auth/utils'
+import { fetchAndSetBanner } from './fetch-and-set-banner'
 
 export const loadAllConfig: (baseUrl: string) => Promise<void> = async (baseUrl) => {
+  const backendUrl = (process.env.REACT_APP_BACKEND || baseUrl)
```

Comment:
Wouldn't it make more sense to extract this into a common available variable?
(Like the `utils/isTestMode`)

Reply:
Tbh I wasn't sure if we need this in more than one places. Most of the time we just need the API url. And this gets set in this method.

Reply:
I moved this var